PyPI sdist tarball is missing 'conftest.py'

**What happened**:
Some tests fail when run from the sdist tarball on PyPI (https://files.pythonhosted.org/packages/18/23/8dea146e14e884e7e0a72738c6bc4ed2ab6939fffbd3ffa32330459861ef/dask-2021.12.0.tar.gz) because of missing fixture, e.g.:

```pytb
____________________________________ ERROR at setup of test_dataframe_shuffle_on_arg[4-on3-False] _____________________________________
file /tmp/portage/dev-python/dask-2021.12.0/work/dask-2021.12.0/dask/dataframe/tests/test_shuffle.py, line 1090
  @pytest.mark.parametrize("ignore_index", [None, True, False])
  @pytest.mark.parametrize(
      "on", ["id", "name", ["id", "name"], pd.Series(["id", "name"])]
  )
  @pytest.mark.parametrize("max_branch", [None, 4])
  def test_dataframe_shuffle_on_arg(on, ignore_index, max_branch, shuffle_method):
E       fixture 'shuffle_method' not found
[...]
```

FWICS this is because the top-level `conftest.py` is not included in the tarball (while tests are).

**What you expected to happen**:

Passing tests ;-).

**Minimal Complete Verifiable Example**:

n/a

**Anything else we need to know?**:

Don't think so.

**Environment**:

- Dask version: 2021.12.0
- Python version: 3.10.0
- Operating System: Gentoo Linux
- Install method (conda, pip, source): source

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -13,3 +13,5 @@ include dask/py.typed
 
 include versioneer.py
 include dask/_version.py
+
+include conftest.py

